### DISCUSSION TITLE
Ebean seems to want to use an Int (presumably from Enum.value()) for an Enum field, despite having an AttributeConverter

### DISCUSSION THREAD

**Discussion by GlenKPeterson** [ebean-orm/ebean#2700] [General]
I originally brought this up under #1777 with a comment I will delete momentarily.

> "I have an enum... I figured I could make an AttributeConverter to store the strings values in the database, but Ebean seems to want to use an Int (presumably from Enum.value())."

Resolved: PEBKAC: I had forgotten to add the AttributeConverter to the list of classes passed to `dbConfig.classes`.  Oops!  Once I did that, it worked a treat.

Rob: you can probably delete your response to my now deleted comment so as not to dilute the original issue.
https://github.com/ebean-orm/ebean/issues/1777#issuecomment-1128150590

**Comment by rbygrave** [comment]
All good.  Nice you found a resolution.  No need to remove comments etc as they might help someone else down the track ...

Cheers, Rob.
